test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Screens/MainScreen.cs

```diff
@@ -29,7 +29,7 @@ public MainScreen(WindowsDriver<WindowsElement> vpnSession)
             this.titleElement = vpnSession.FindElementByClassName("HeroText");
             this.subtitleElement = vpnSession.FindElementByClassName("HeroSubText");
             this.settingButton = vpnSession.FindElementByName("Settings");
-            this.vpnSwitch = vpnSession.FindElementByName("Toggle VPN");
+            this.vpnSwitch = vpnSession.FindElementByName("Toggle");
             this.serverListButton = vpnSession.FindElementByAccessibilityId("ConnectionNavButton");
             this.deviceListButton = vpnSession.FindElementByName("My devices");
```

Comment:
It looks like some elements here have been identified by class name, name or accessibility id.  Is this there any reason why different methods were used?  Are we missing any element properties in the app that will help make this easier.

Reply:
Some of elements missing name properties, some of them have duplicate name properties. I need to use both name and accessibility Id to identify all elements.